### PR TITLE
upd: removed tracking auto-generated files and vscode settings

### PR DESCRIPTION
**Pull request by NguyenDuck**
- `dist` folder very conflict on merge
- `deno.lock` file generated depend on `deno.json`
- vscode setting `editor.formatOnSave` make the json files break, inline json array or space at start, end of array items

Not format
```jsonc
{
    "array_name": ["item1", "item2"]
}
```

Formated
```jsonc
{
    "array_name": [
        "item1",
        "item2"
    ]
}
```

Formated ( `json.format.keepLines` enabled )
```jsonc
{
    "array_name": [ "item1", "item2" ]
}
```

### CODE REVIEW

File: deno.lock
Comment (ThomasOrs) [inline deno.lock:1]: Lock files are important in open-source projects to ensure consistent versioning between users.

File: .gitignore [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
+dist
 
 .env
 packages.zip
 dataPackage.ts
-scripts/vanillaData/raw
\ No newline at end of file
+scripts/vanillaData/raw
+
+deno.lock
```
Comment (ThomasOrs) [inline .gitignore:10]: Lock files are important in open-source projects to ensure consistent versioning between users.

**Review by ThomasOrs — CHANGES_REQUESTED** [review]
See comments

**Comment by outercloudstudio** [comment]
Deno.lock can not be used with this project as this project uses sky pack which appears to change the hash of the dependencies without control preventing the lock file from being useful.

**Comment by outercloudstudio** [comment]
Closing this without merging since the only changes needed from this is fixing dist folder conflict. The other changes should not be made.

**Pull request closed without merging**